operator/option/config.go

```diff
@@ -413,6 +416,7 @@ func (c *OperatorConfig) Populate() {
 	c.IdentityGCRateInterval = viper.GetDuration(IdentityGCRateInterval)
 	c.IdentityGCRateLimit = viper.GetInt64(IdentityGCRateLimit)
 	c.IdentityHeartbeatTimeout = viper.GetDuration(IdentityHeartbeatTimeout)
+	c.NodeGCInterval = viper.GetDuration(NodesGCInterval)
 	c.NodesGCInterval = viper.GetDuration(NodesGCInterval)
```

Comment:
@sayboras I've moved this new line right next to the "old" `c.NodesGCInterval` (notice the Nodes plural in the name) which is deprecated in v1.11 and recycled in v1.12 (the upstream commit). Is it OK to have this behaviour introduced in v1.10 (basically piggy backing on the old CLI flag)?

Also note that `c.NodesGCInterval` is set twice in this func (second time new line 432), but I don't think it matter.